Package fails to build on OS X build machine

Building this package fails on the OS X build machine, see error below. Seems to refer to `node-sass`. The package builds fine on my local OS X machine, as well as the Linux and Windows ones.

Any ideas @stubailo, @slava, @avital ?

```
build:~ meteor$ meteor publish-for-arch lookback:emails@0.4.2
lookback:emails: updating npm dependencies -- node-sass, juice...
npm WARN package.json packages-for-meteor-smartpackage-1hrfjrq@0.0.0 No
description
npm WARN package.json packages-for-meteor-smartpackage-1hrfjrq@0.0.0 No
repository field.
npm WARN package.json packages-for-meteor-smartpackage-1hrfjrq@0.0.0 No
README data
npm WARN deprecated CSSselect@0.4.1: the module is now available as
'css-select'
npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'

You have not agreed to the Xcode license agreements, please run xcodebuild
standalone from within a Terminal window to review and agree to the Xcode
license agreements.

In file included from ../src/libsass/context.cpp:20:
In file included from ../src/libsass/parser.hpp:12:
In file included from ../src/libsass/prelexer.hpp:5:
../src/libsass/lexer.hpp:148:26: error: cannot use type 'void' as a
range
for (prelexer mx : { mxs... }) {
^
../src/libsass/lexer.hpp:160:26: error: cannot use type 'void' as a
range
for (prelexer mx : { mxs... }) {
^
2 errors generated.
make: *** [Release/obj.target/libsass/src/libsass/context.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit
(/private/var/folders/s3/5fl5zcbn3jbfdnnyhhgxxvvm00007b/T/package-source1eggdgl/lookback_emails/.npm/package-new-84fttv/node_modules/node-sass/node_modules/pangyp/lib/build.js:271:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit
(child_process.js:820:12)
gyp ERR! System Darwin 11.4.2
gyp ERR! command
"/Users/meteor/.meteor/packages/meteor-tool/.1.1.3.1wysac9++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
"/private/var/folders/s3/5fl5zcbn3jbfdnnyhhgxxvvm00007b/T/package-source1eggdgl/lookback_emails/.npm/package-new-84fttv/node_modules/node-sass/node_modules/pangyp/bin/node-gyp"
"rebuild" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags="
"--libsass_library="
gyp ERR! cwd
/private/var/folders/s3/5fl5zcbn3jbfdnnyhhgxxvvm00007b/T/package-source1eggdgl/lookback_emails/.npm/package-new-84fttv/node_modules/node-sass
gyp ERR! node -v v0.10.36
gyp ERR! pangyp -v v2.2.1
gyp ERR! not ok
Build failed

npm ERR! node-sass@3.2.0 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@3.2.0 postinstall script.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 11.4.2
npm ERR! command
"/Users/meteor/.meteor/packages/meteor-tool/.1.1.3.1wysac9++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
"/Users/meteor/.meteor/packages/meteor-tool/.1.1.3.1wysac9++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm"
"install"
npm ERR! cwd
/private/var/folders/s3/5fl5zcbn3jbfdnnyhhgxxvvm00007b/T/package-source1eggdgl/lookback_emails/.npm/package-new-84fttv
npm ERR! node -v v0.10.36
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! not ok code 0

=> Errors while initializing project:

While building package lookback:emails:
error: couldn't install npm packages from npm-shrinkwrap
```

I got the same, building on OSX locally

```
pm ERR! Error: EACCES, mkdir '/Users/satya/.npm/minimist/1.1.1'
npm ERR!  { [Error: EACCES, mkdir '/Users/satya/.npm/minimist/1.1.1']
npm ERR!   errno: 3,
npm ERR!   code: 'EACCES',
npm ERR!   path: '/Users/satya/.npm/minimist/1.1.1',
npm ERR!   parent: 'meow' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! System Darwin 14.0.0
npm ERR! command
"/Users/satya/.meteor/packages/meteor-tool/.1.1.3.1ekqsng++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
"/Users/satya/.meteor/packages/meteor-tool/.1.1.3.1ekqsng++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "install"
"node-sass@3.2.0"
npm ERR! cwd /Users/satya/Development/meteor-packages/lookback:emails/.npm/package-new-1xru2zs
npm ERR! node -v v0.10.36
npm ERR! npm -v 1.4.28
npm ERR! path /Users/satya/.npm/minimist/1.1.1
npm ERR! code EACCES
npm ERR! errno 3
npm ERR! stack Error: EACCES, mkdir '/Users/satya/.npm/minimist/1.1.1'
npm ERR! not ok code 0

 => Errors while adding packages:

While building package satya:emails:
error: couldn't install npm package
```

--- a/emails.coffee
+++ b/emails.coffee
@@ -33,8 +33,6 @@ Mailer =
 
 # ## Deps
 #
-# Use `juice` for inlining CSS into the templates.
-juice = Npm.require 'juice'
 Utils = share.MailerUtils
 
 # ## Template helpers
@@ -349,16 +347,16 @@ MailerClass = (options) ->
       # Typically `/emails/preview/myEmailTemplate`.
       # Do some formatting. The `path` should be the prefix, followed by
       # the type (`preview` or `send`). So it could look like
-      # 
+      #
       #    /emails/preview/sampleTemplate/:param
-      # 
+      #
       # Also capitalize the first character in the template name for
-      # the name of the route, so it will look like `previewSample` for a 
+      # the name of the route, so it will look like `previewSample` for a
       # template named `sample`.
       path = "#{settings.routePrefix}/#{type}" + template.route.path
       name = Utils.capitalizeFirstChar(template.name)
       routeName = "#{type}#{name}"
-      
+
       Utils.Logger.info "Add route: [#{routeName}] at path /" + path, TAG
 
       Router.route routeName,

--- a/package.js
+++ b/package.js
@@ -7,11 +7,6 @@ Package.describe({
   git: 'https://github.com/lookback/meteor-emails.git'
 });
 
-Npm.depends({
-  'node-sass': '3.2.0',
-  'juice': '1.0.0'
-});
-
 Package.onUse(function(api) {
 
   api.versionsFrom('1.0.4');
@@ -21,6 +16,8 @@ Package.onUse(function(api) {
     'underscore',
     'coffeescript',
     'email',
+    'sacha:juice',
+    'chrisbutler:node-sass',
     'iron:router@1.0.7',
     'meteorhacks:ssr@2.1.2'
   ], where);

--- a/utils.coffee
+++ b/utils.coffee
@@ -2,7 +2,6 @@
 
 fs = Npm.require 'fs'
 path = Npm.require 'path'
-sass = Npm.require 'node-sass'
 
 # This package assumes that assets (templates, SCSS, CSS ..) are
 # stored in the `private` directory. Thanks to that, Meteor won't

